scripts/build/README.md

```diff
@@ -79,8 +79,6 @@ yarn build --report=all
 yarn build --report=stdout --report=text --report=html
 ```
 
-**`--report` equals to `--report=html`**
```

Comment:
I don't know how to make it happen with `node:utils.parsArgs`

Reply:
It can't work anymore, you can use `--report=all`